src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    /**
+     * Returns a printable string representation of the data type of a person
+     */
+    String getPrintableString();
```

Comment:
did you miss the second part of the LO? you need to add a `getPrintableString` in `Person` class to use the other getPrintableString methods

Reply:
Fixed 4462f6afca3e6188ff3eb818e05fbbcd6772a3c9